app/controllers/resources_controller.rb

```diff
@@ -11,6 +11,13 @@ def index
     options = params.permit(*allowed_params)
       .slice(*allowed_params).to_h.symbolize_keys
 
+    if Rails.application.config.conjur_config.api_resource_list_limit.positive?
+      options[:limit] = Rails.application.config.conjur_config.api_resource_list_limit.to_s unless options[:limit]
+
+      raise ApplicationController::UnprocessableEntity, "Limit value should not exceed #{Rails.application.config.conjur_config.api_resource_list_limit}" \
+      unless options[:limit].to_i <= Rails.application.config.conjur_config.api_resource_list_limit
```

Comment:
Given how long this line is, please use 
```ruby
unless ....
   raise
end
```

instead.

Reply:
done